zerver/lib/push_notifications.py

```diff
@@ -456,9 +457,18 @@ def get_apns_payload(message):
         'custom': {
             'zulip': {
                 'message_ids': [message.id],
+                'sender_email': message.sender.email,
```

Comment:
Do we want to encode the sender ID rather than the email?  That's more future-proof, since email addresses change.

Reply:
For the private narrows in the app, we haven't migrated to use the id's yet. 

But we can start from here @borisyankov ?

Reply:
Let's just send both sender_id and sender_email and the app can migrate when it has a chance.  Seems cleaner that way.

Reply:
👍